src/tests/functional/plugin/shared/src/snippets/matmul.cpp

```diff
@@ -42,38 +58,57 @@ void MatMul::SetUp() {
 }
 
 void MatMul::init_subgraph(const std::vector<ov::element::Type>& types) {
-    auto f = ov::test::snippets::MatMulFunction(inputDynamicShapes, types);
+    auto f = ov::test::snippets::MatMulFunction(inputDynamicShapes, types, matmul_type);
     function = f.getOriginal();
+    filter_shape_info(f.get_constant_input_idces());
```

Comment:
Maybe does it make sense to move this method call to `SetUp` after the call `init_subgraph(elem_types);`? Wha do you think?

Reply:
Not bad option, I think. We will need to return snippets function from this virtual method, but I think this is not problem

Reply:
Done, thanks